app/src/org/commcare/utils/FileUtil.java

```diff
@@ -140,7 +195,6 @@ public static void copyFile(File oldPath, File newPath) throws IOException {
         } else {
             Log.e(LOG_TOKEN, "Source file does not exist: " + oldPath.getAbsolutePath());

```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/a72c848e609bc036eec2b6ba409562525ed25893/app/src/org/commcare/utils/FileUtil.java#L196"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>